src/libappimage/libappimage.c

```diff
@@ -2136,7 +2137,7 @@ int appimage_get_type(const char* path, bool verbose)
         fseek(f, 8, SEEK_SET);
         fread(buffer, 1, 3, f);
         fclose(f);
-        if((buffer[0] == 0x41) && (buffer[1] == 0x49) && (buffer[2] == 0x01)){
+        if(match_type_1_magic_bites(buffer)){
```

Comment:
I don't see a point in extracting this into a function.

Reply:
Now you know what that the conditions inside the if mean, just by reading it.